sympy/polys/ring_series.py

```diff
@@ -753,7 +753,7 @@ def rs_diff(p, x):
     for expv in p:
         if expv[n]:
             e = monomial_ldiv(expv, mn)
-            p1[e] = p[expv]*expv[n]
+            p1[e] = R.domain_new(p[expv]*expv[n])
```

Comment:
After the `PolyElement` for a particular series is computed , it is then transformed to an `expr` which is then displayed as the result. In doing so, the method `as_expr_dict()` is called : 
```
def as_expr_dict(self):
        to_sympy = self.ring.domain.to_sympy
        return {monom: to_sympy(coeff) for monom, coeff in self.iterterms()}
```
Here for field QQ , the `to_sympy` method uses attributes of `PythonRational` namely `numerator` and `denominator`.
```
def to_sympy(self, a):
        """Convert `a` to a SymPy object. """
        return SymPyRational(a.numerator, a.denominator)
```
An error obviously occurs if we do not coerce to the datatype of the concerned field : 
```
Traceback (most recent call last):
  File "test.py", line 11, in <module>
    print(rs_series(log(2 + a),a,10).as_expr())
  File "/home/arif7/sympy/sympy/sympy/polys/rings.py", line 609, in as_expr
    return expr_from_dict(self.as_expr_dict(), *symbols)
  File "/home/arif7/sympy/sympy/sympy/polys/rings.py", line 613, in as_expr_dict
    return {monom: to_sympy(coeff) for monom, coeff in self.iterterms()}
  File "/home/arif7/sympy/sympy/sympy/polys/rings.py", line 613, in <dictcomp>
    return {monom: to_sympy(coeff) for monom, coeff in self.iterterms()}
  File "/home/arif7/sympy/sympy/sympy/polys/domains/pythonrationalfield.py", line 30, in to_sympy
    return SymPyRational(a.numerator, a.denominator)
AttributeError: 'Rational' object has no attribute 'numerator'
```
Actually such coercions probably need to be done in many other helper functions as well.